micromasters/views.py

```diff
@@ -0,0 +1,15 @@
+"""
+micromasters views
+"""
+from django.contrib.auth.views import redirect_to_login
+from django.urls import reverse
+
+
+def cms_signin_redirect_to_site_signin(request):
+    """Redirect wagtail admin signin to site signin page"""
+    return redirect_to_login(reverse('wagtailadmin_home'), login_url="/")
+
+
+def cms_password_reset_redirect_to_site_signin(request):
+    """Redirect wagtail admin password reset to site signin page"""
+    return redirect_to_login(reverse('wagtailadmin_home'), login_url="/")
```

Comment:
shouldn't this redirect to forgot password page in edx?

Reply:
We normally refer to `LOGIN_URL ='/'` for it, We can't directly add edx link because of not allowed hosts which are referring to other hosts.

Plus if use that it will redirect to `/cms` after you login.